### PR TITLE
Retrieve Piped name from control-plane

### PR DESCRIPTION
**Pull request by ono-max**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes part of #2514

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/app/api/grpcapi/piped_api.go [hunk 1]
```diff
@@ -149,7 +149,19 @@ func (a *PipedAPI) ReportPipedMeta(ctx context.Context, req *pipedservice.Report
 			return nil, status.Error(codes.Internal, "failed to update the piped metadata")
 		}
 	}
-	return &pipedservice.ReportPipedMetaResponse{}, nil
+
+	var piped *model.Piped
+	piped, err = a.pipedStore.GetPiped(ctx, pipedID)
```
Comment (nghialv) [inline pkg/app/api/grpcapi/piped_api.go:154]: You can use this helper to find the piped. It is also handling the error for us.

``` golang
	piped, err := getPiped(ctx, a.pipedStore, pipedID, a.logger)
        if err != nil {
		return nil, err
	}
```

Comment (ono-max) [reply pkg/app/api/grpcapi/piped_api.go]: 902afd4

**Review by khanhtc1202 — APPROVED** [review]

**Pull request merged**